guide/english/css/css3-media-queries/index.md

```diff
@@ -3,20 +3,15 @@ title: CSS3 Media Queries
 ---
 ## CSS3 Media Queries
 
-Media Queries allow you to have different styles for different devices/screen sizes. Their introduction in CSS3 has greatly eased the building
-of responsive webpages.
+Media queries allow you to design a website differently for different devices/screen sizes. Their introduction in CSS3 has greatly eased the building of responsive webpages.
 
-The best approach when designing a responsive website is to think mobile first; meaning that you create your page starting with the design and content
-of the mobile version. You may think that with some scalable sizes ( %, vw or vh ), your page will adapt perfectly to any device. But it will not. Maybe
-for some very basic design, but certainly not for more common or complex pages!
+The best approach when designing a responsive website is to think <em>mobile first</em> - meaning that you design the mobile version of your site first and scale up for larger devices. Using relative units of size (like %, vw or vh) will not guarantee your page adapts perfectly to any device, especially for complex layouts! Media queries let you specify <em>breakpoints</em> where different styles will be applied.
 
-When designing your page for smaller devices, you will focus on the main content. On a bigger screen, you will have to adapt some font-sizes, margins,
-paddings and so on in order to keep your site comfortable and readable, but you will also want/need to add more content, the ones you did not judge
-fundamental, and fill in the space created by the screen size.
+When designing your page for smaller devices, you should focus on the main content. On a bigger screen, you will want to adjust font sizes, margins, padding, etc. to keep your site readable, but you may also want to add secondary content to fill in the space created by the screen size.
 
-The thought process should be:
+The thought process for responsive design should be:
 1. Which content to show?
-2. How to layout the page?
+2. How to lay out the page?
```

Comment:
Instead of structuring this line as you have done, we could instead write it as:
`How to create the layout of the page` this removes the problem of `lay out` vs `layout` as both conflict in their own right.

Reply:
I like this suggestion, will fix.